src/histolab/filters/image_filters.py

```diff
@@ -0,0 +1,700 @@
+from PIL import ImageOps
+
+from src.histolab.filters import image_filters_functional as F
+
+
+class Compose(object):
+    """Composes several filters together.
+
+    Parameters
+    ----------
+    filters : list of Filters
+        List of filters to compose
+    """
+
+    def __init__(self, filters):
+        self.filters = filters
+
+    def __call__(self, img):
+        for f in self.filters:
+            img = f(img)
+        return img
+
+
+class Invert(object):
+    """Invert an image."""
```

Comment:
improve the `docstring` here, it's not enough to understand what this filter does